### PR TITLE
Prepare release 0.4.6

### PR DESCRIPTION
**Pull request by syssi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,71 @@
 # Change Log
 
+This release adds support for the following new devices:
+
+* Xiaomi Air Quality Monitor S1 \(cgllc.airmonitor.s1\)
```
Comment (rytilahti) [inline CHANGELOG.md]: The parentheses are escaped here but not some lines below.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,71 @@
...
+
+* Air Conditioning Companion: Add particular swing mode values of a chigo air conditioner
+* Air Humidifier: Handle poweroff exception on set\_mode
+* Chuangmi IR controller: Add indicator led support
+* Chuangmi IR controller: Add discovery of the Xiaomi IR remote 2gen \(chuangmi.remote.h102a03\)
+* Chuangmi Plug: Fix set\_wifi\_led cli command
+* Vacuum: Add state 18 as "segment cleaning"
+* Device: Add easily accessible properties to DeviceError exception
+* Always import DeviceError exception
+* Require click version \>=7
```
Comment (rytilahti) [inline CHANGELOG.md]: Escaping also here (and above for `_`).

**Comment by shaymdev** [comment]
I'm following this in anticipation of segment cleaning for the s6 vacuum (and hoping for rapid home assistant upgrade to your new 0.4.6). Anything I can do to help out?

**Comment by rytilahti** [comment]
There has been some more new changes since creation of this PR, so the changelog needs to be updated before the release.

If you want to help, I think the best way to do that would be to see how to integrate segment cleaning on homeassistant's end against master branch of python-miio. That way it'd be fast to fix before releasing a new version and bumping the homeassistant requirement.

**Pull request merged**